client/src/components/Header.jsx

```diff
@@ -0,0 +1,7 @@
+
+
+const Header = () => {
+  return <h2>Expense Tracker</h2>;
```

Comment:
Since this component is called Header, it would be nice to have a `header` tag wrapped around the `h2`